scripts/list-pull-requests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Fetches pull request refs and prints commits
+# for the given range grouped by pull request
+
+UPSTREAM_REMOTE="upstream"
```

Comment:
Just a note, i don't have "upstream", but have "origin" because i'm using status-im/status-react repository directly. So i end up chaning this line to `UPSTREAM_REMOTE="origin"`
to make it work.

Reply:
@jwagener Any idea how we could handle both cases?

Reply:
I changed it to use a separate remote name to fetch the PR refs. Now it should be completely independent from the developer remote setup.